client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -318,6 +319,9 @@ const InvestigationTable: React.FC = (): JSX.Element => {
         return filterIcon
     }
 
+    const isInvestigationRowClickable = (investigation: InvestigationTableRow) =>
```

Comment:
why not pass just mainStatus as a parameter instead all the row?

Reply:
I think that sending the whole row is better as considering possible future checks